core/core-spacemacs-buffer.el

```diff
@@ -1237,12 +1237,12 @@ SEQ, START and END are the same arguments as for `cl-subseq'"
 (defun spacemacs-buffer//insert-recent-files (list-size)
   (unless recentf-mode (recentf-mode))
   (setq spacemacs-buffer//recent-files-list
-        (cl-delete-if (lambda (x)
-                        (or (when (and (bound-and-true-p org-directory) (file-exists-p org-directory))
-                              (member x (directory-files org-directory t)))
-                            (when (bound-and-true-p org-agenda-files)
-                              (member x (mapcar #'expand-file-name org-agenda-files)))))
-                      recentf-list))
+        (let ((agenda-files (mapcar #'expand-file-name (org-agenda-files))))
```

Comment:
Please use
```elisp
(cl-delete-if (lambda (x)
                (or (when (and (bound-and-true-p org-directory) (file-exists-p org-directory))
                      (member x (directory-files org-directory t)))
                    (when (fboundp 'org-agenda-files)
                      (member x (mapcar #'expand-file-name (org-agenda-files))))))
              recentf-list))
```

This added a `fboundp` test so it won't err when `org-agenda` is disabled by the user.

Reply:
Done!